libraries/src/Editor/Editor.php

```diff
@@ -255,7 +255,7 @@ public function initialise()
 
 		foreach ($results as $result)
 		{
-			if (trim($result))
+			if (trim((string) $result))
```

Comment:
```suggestion
			if (!is_null($result) && trim($result))
```

That should do the same trick without casing anything to string but checking whether we have a null value for whatever reason.

Reply:
I agree. @beat Could you accept this suggestion? Thanks in advance.

Reply:
Yup, it's cleaner and equivalent, also have tested it. So accepting the suggestion. Thank you @zero-24 and @richard67 !